src/WebJobs.Extensions.EasyTables/EasyTableAttribute.cs

```diff
@@ -37,5 +38,21 @@ public sealed class EasyTableAttribute : Attribute
         /// May include binding parameters
         /// </summary>
         public string Id { get; set; }
+
+        /// <summary>
+        /// Optional. A string value indicating the app setting to use as the Azure Mobile App Uri, if different
+        /// than the one specified in the <see cref="EasyTablesConfiguration"/>.
+        /// </summary>
+        public string MobileAppUri { get; set; }
+
+        /// <summary>
+        /// Optional. A string value indicating the app setting to use as the Azure Mobile App Api Key, if different
+        /// than the one specified in the <see cref="EasyTablesConfiguration"/>.
+        /// This value affects this parameter as follows:
+        /// - If it is null, the ApiKey value from <see cref="EasyTablesConfiguration"/> is used.
+        /// - If it is equal to string.Empty, no Api Key is used.
+        /// - If it is any other string, the value of that string is used as the Api Key.
+        /// </summary>
+        public string ApiKey { get; set; }
```

Comment:
Thoughts on this? I needed a way to let the parameter itself say "I don't want any api key" -- but also need a default way (null) to say "just use the global one"